ios/Podfile.lock

```diff
@@ -1055,4 +1055,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 9fd323641c96f6bf98b309066332c3ff95b9cf15
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.10.1
```

Comment:
I think you commited this IOS files by mistake. 
You should update your cocoapods to 1.11.2